hal/src/main/native/athena/HAL.cpp

```diff
@@ -594,6 +594,14 @@ HAL_Bool HAL_Initialize(int32_t timeout, int32_t mode) {
   });
 
   if (!SetupNowRio()) {
+    fmt::print(stderr,
+               "Failed to run SetupNowRio(). This is a fatal error. The "
+               "process is being terminated.\n");
+    std::fflush(stdout);
+    // Attempt to force a segfault to get a better java log
+    *(int*)0 = 0;
```

Comment:
This should use a reinterpret cast instead, right? If not it needs a no lint as its failing the workflow

Reply:
Yea, it should use reinterpret_cast.